llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -5944,8 +5953,21 @@ bool SPIRVToLLVM::translate(ExecutionModel entryExecModel, const char *entryName
   m_enableGatherLodNz =
       m_bm->hasCapability(CapabilityImageGatherBiasLodAMD) && entryExecModel == ExecutionModelFragment;
 
-  m_dbgTran.createCompileUnit();
-  m_dbgTran.addDbgInfoVersion();
+  // Find the compile unit first since it might be needed during translation of
+  // debug intrinsics.
+  MDNode *compilationUnit = nullptr;
+  for (SPIRVExtInst *EI : m_bm->getDebugInstVec()) {
```

Comment:
EI  -> ei
In SPIRV-reader.cpp, we follow the LLPC style (lowercase-initial). The naming will be change to extInst or spvExtInst in the future. Currently, you can keep it consistent with upstreaming codes.

Reply:
Acknowledged. I copy-pasted from upstream and made only the necessary changes to integrate into LLPC. My thinking is we'd be able to keep at least the debug info implementation in sync with upstream by making this copy match as closely as possible (I'll work out a way to do this more easily / frequently in the next few weeks)